src/main/java/com/lpvs/controller/GitHubWebhooksController.java

```diff
@@ -42,7 +42,7 @@ public class GitHubWebhooksController {
     ApplicationContext applicationContext;
 
     @PostConstruct
-    private void setProps() {
+    public void setProps() {
```

Comment:
Any reason to swap the type of `setProps` from private to public? Just for my double check with you. 🙏

Reply:
It's a typical problem that PostConstruct method is not calling in unit test when we create service or controller manually.
For example, https://stackoverflow.com/questions/50716045/postconstruct-not-being-called-in-junits

So, to test this method properly, we have to change private to public.

Reply:
Thank you for the prompt clarification. One more question. Any alternative approach at this moment?

Reply:
No, because for unit test we have to create the controller via `new` operator. PostConstruct will not be called automatically.